app/scripts/metamask-controller.js

```diff
@@ -873,6 +873,13 @@ export default class MetamaskController extends EventEmitter {
                   await this.approvalController.success({
                     flowToEnd: addAccountApprovalId,
                     message: t('snapAccountCreated'),
+                    header: [
+                      {
+                        key: 'snapHeader',
+                        element: 'SnapAuthorshipHeader',
+                        properties: { snapId: origin },
```

Comment:
You're missing the 'props' key in the template. only the items passed to the props key are passed through to the component specified in 'element' (along with key )